apps/trading-e2e/src/integration/withdrawals.feature

```diff
@@ -0,0 +1,67 @@
+Feature: Withdrawals
+
+  Scenario: Can prepare a withdrawal
+    Given I navigate to withdrawals page
+    And I connect eth wallet
+    And I connect vega wallet
+    And I can see the withdrawals warning info
+    And I can see the from ethereum address field is automatically populated with current connected key
+    And I select asset 'tBtc'
+    And I input '100' tokens to withdraw
+    And the withdraw button is enabled
+    When I click the withdraw button
+    Then I am redirected to the finalise withdrawals page
```

Comment:
This has changed a bit and the now the full withdrawal happens all on the form page. As a first pass I think we should just assert here that the transaction dialog shows with the correct txHash